test/security/PauseManager.t.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.16;
+
+import "forge-std/Test.sol";
+import "../../contracts/security/PauseManager.sol";
+import "../../contracts/mocks/tokens/MockTestnetToken.sol";
+
+contract PauseManagerTest is Test {
+    address internal _user1;
+    address internal _user2;
+
+    function setUp() public {
+        _user1 = vm.rememberKey(2);
+        _user2 = vm.rememberKey(1);
+    }
+
+    function testShouldAddGuardian() public {
+        // given
+        assertFalse(PauseManager.isGuardian(_user1));
```

Comment:
We generally use asserts only in `then` section

Reply:
ok